homeassistant/components/radio_browser/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""The Radio Browser integration."""
+from __future__ import annotations
+
+from radios import RadioBrowser, RadioBrowserError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import __version__
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Radio Browser from a config entry.
+
+    This integration doesn't set up any enitites, as it provides a media source
+    only.
+    """
+    session = async_get_clientsession(hass)
+    radios = RadioBrowser(session=session, user_agent=f"HomeAssistant/{__version__}")
```

Comment:
don't we already have a user agent in our default websession ?

Reply:
Yes, but as the API requires a custom user agent to be set (usage requirement),
I've enforced the library require it on init as well.